### DISCUSSION TITLE
Programmazione attività / inserimento programmazione scolatica

### DISCUSSION THREAD

**Discussion by Forrest-It** [iisgiua/giuaschool#380] [Richieste di aiuto]
Buon pomeriggio a tutti,

premesso che sono un genitore di una bambina che frequenta le elementari che per poter seguire meglio le attività scolatica di mia figlia ha proposto all'istituto che frequenta -  a titolo gratuito - l'utilizzo di un registro elettonico open source:  ho  installato giua@school e vorrei, prima di proporne l'uso, capire alcune funzionalità. In particolare non riesco a capire come inserire la programmazioni delle attività per una singola classe: immaginavo che, utilizzando l'accesso docente, fosse possibile inserire una data attività per un dato giorno e una dta ora : evidentemente la filosofia di funzionamento di giuia@school e ben diversa.

Vi scrivo per chiedere aiuto , per capire meglio la filosofia di funzionamento di giuia@school : ho letto i vari manuali disponibili ma non ho trovato risposta in merito alla programmazione delle attività scolastiche per le varie classi. 

Grazie mille

Tommmaso

**Comment by Lucaccc** [comment]
Buon pomeriggio, le rispondo io che utilizzo giuaschool da qualche anno.
Per quanto riguarda l'inserimento dei piani di lavoro/programma finale, questi si inseriscono nell'apposita aerea (lato docente): basta cliccare sul menù "documenti" e seguire le indicazioni date.
Per la programmazione delle attività, basta che clicca su agenda e inserisce compiti, attività, verifiche (con gli appositi spazi).
Per ulteriori annotazioni del docente relative a delle attività di un determinato giorno, basta andare nella pagina della classe, cliccare sull'area delle annotazioni e successivamente renderle visibili anche a studenti e genitori.

**Reply by Forrest-It** [reply]
Grazie mille per la celere risposta .

per la programmazione delle attività,  nell'agenda non trovo il tasto attività ma solo "aggiungi verifica" e aggiungi compiti" :  c'è da agire lato amministratore o lato dirigente scolastico  o altro?

Grazie mille

Tommaso

**Reply by Lucaccc** [reply]
Se c'è un'attività particolare per la classe, questa può essere inserita dal lato staff (nella sezione per l'inserimento delle circolari). In questo modo dovrebbe apparire sul calendario.
In alternativa, può utilizzare la sezione annotazioni che le ho detto prima.

**Reply by trinko** [reply]
Premetto che il registro è pensato per una scuola superiore e pertanto potrebbe presentare dei limiti per una scuola elementare.

Comunque, dipende da cosa intendi per attività da programmare. 

Se è un'attività che riguarda alunni o genitori, che per il giorno X si prevede che facciano qualcosa, si può usare l'inserimento di un COMPITO dall'agenda, inserito direttamente dal docente.

Se è un'attività che coilvolge diverse classi ed è organizzata dalla scuola e non dal singolo docente della classe, allora l'inserimento deve essere fatto dal dirigente o dal suo staff, tramite la pagina **STAFF > AVVISI**. Questa procedura genera sia un messaggio di avviso che un pulsante visibile nell'agenda.

Se invece intendi come il docente inserisce l'organizzazione della sua attività didattica, allora, come ti è stato detto, il docente inserisce il piano delle attività a inizio anno, tramite la pagina **DOCUMENTI > PIANI DI LAVORO**. Poi il docente inserisce l'attività svolta giorno per giorno nelle lezioni.

**Reply by Forrest-It** [reply]
Premetto che non sono un insegnante ed è possibile che abbia usato dei termini sbagliati. 

Immaginavo che andava inserito una programmazione giornaliera dove andava indicata, per ogni classe, l'insegnamento previsto a una data ora e a quel punto, sul programmazione preeinseita, interveniva il docente inserendo l'argomento e i dettagli: il genitorè così avrebbe la possibilità di sapere nei dettagli le attività svolte dal figlio nelle varie materie .
 
Leggendo il manuale del DOCENTE vedo l'immagine allegata (registro_manuale), relativamente al registo :  a una data ora, sono popolati i campi relativi materia/docenti/argomenti ed è possibile modificare o cancellare il contenuto.

Nel caso dell'instazione fatta da me, come si vede dalla seconda immagine allegata ( registro_installazione) tutti i campi sono vuoti e non è possibile agire in alcun modo.

Nella agenda, come si vede dalla terza immagine allegata ( agenda) c'è solo la possibilità di inserire "aggiungi compiti" e "aggiungi verifica" : mi aspettavo di trovare quì  "aggiungi attività".

Lo scopo fondamentale dell'utilizzo del registro elettronico per la scuola frequentata da mia figlia ( elementare ) è quella di dare la possibilità agli insegnati di informare tramite lo stesso degli argomenti trattati ogni giorno nelle varie materie, con la finalità di dare ai genitori la possibilità di seguire con più efficacia i propri figli.

Credo che, nell'inserire i vari dati di prova nell'applicazione, abbia omesso qualcosa o che non sia possibile inserire attività di domenica. 

Grazie mille 

Tommaso
![registro_manuale](https://user-images.githubusercontent.com/73697572/193472034-912ff595-7208-4495-9e58-64f3c416aa58.jpg)
![agenda](https://user-images.githubusercontent.com/73697572/193472035-e4a61361-a633-4a32-9247-709ba61edafe.jpg)
![registro_installazione](https://user-images.githubusercontent.com/73697572/193472037-0685c582-8c44-4905-9a1d-ad53838da931.jpg)

**Reply by Lucaccc** [reply]
Prima di tutto, non è possibile inserire la firma per una giornata futura (come nel suo caso a lunedì).
Se va nelle giornate precedenti (esempio venerdì) le da la stessa schermata?

In ogni caso, no: non va inserita una programmazione che associa ogni materia a una data e ora in una determinata classe.
I docenti firmano l'ora che è prevista dall'orario scolastico.

**Reply by Forrest-It** [reply]
Tutto chiaro : ho visto che nelle gionati precedenti è possibile inserire le attività.

Grazie mille
